<launch>
<include file="$(find yumi_description)/launch/display.launch" />
<node name="kdl_kinematics" type="kdl_kinematics" pkg="controller" />
<node name="yumiCartesianVelocityPlanner" type="cartesianVelocityPlanner.py" pkg="controller" output="screen"/>
</launch>